### PR TITLE
common: automate setting up online repositories for ceph deployments …

### PR DESCRIPTION
**Pull request by leseb**
…on debian nodes

This commits automates the process of setting up online repositories for
Red Hat Ceph Storage on Debian nodes. The manual steps are currently
described here:
https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/2/html/installation_guide_for_ubuntu/prerequisites#online_repositories

If you are an RHCS customer and run a Debian based system you can now
access package through the Red Hat CDN.
For this set: ceph_rhcs and ceph_rhcs_cdn_install to true. Then set your
customer credentials in ceph_rhcs_cdn_debian_repo. Replace
customername:customerpasswd with your details.

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1434175
Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by leseb** [comment]
jenkins test jewel-ansible2.2-docker_dmcrypt_journal_collocation

**Comment by leseb** [comment]
jenkins test  jewel-ansible2.2-update_docker_cluster

**Comment by leseb** [comment]
jenkins test luminous-ansible2.2-purge_cluster

**Comment by leseb** [comment]
jenkins test luminous-ansible2.2-update_cluster

**Comment by leseb** [comment]
jenkins test luminous-ansible2.2-purge_docker_cluster

**Comment by leseb** [comment]
jenkins test jewel-ansible2.2-update_docker_cluster

**Comment by leseb** [comment]
jenkins test jewel-ansible2.2-update_docker_cluster

**Comment by leseb** [comment]
jenkins test jewel-ansible2.2-update_cluster

**Comment by leseb** [comment]
jenkins test this please

File: roles/ceph-common/tasks/pre_requisites/prerequisite_rhcs_cdn_install_debian.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+- name: determine if node is registered with subscription-manager
```
Comment (leseb) [inline roles/ceph-common/tasks/pre_requisites/prerequisite_rhcs_cdn_install_debian.yml]: I need to remove this

**Comment by andrewschoen** [comment]
jenkins test jewel-ansible2.2-dmcrypt_journal

**Comment by andrewschoen** [comment]
jenkins test jewel-ansible2.2-purge_docker_cluster

**Comment by andrewschoen** [comment]
jenkins test jewel-ansible2.2-update_cluster

**Comment by andrewschoen** [comment]
jenkins test luminous-ansible2.2-bluestore_cluster

**Comment by andrewschoen** [comment]
luminous-ansible2.2-centos7_cluster
luminous-ansible2.2-purge_docker_cluster

**Comment by andrewschoen** [comment]
luminous-ansible2.2-update_docker_cluster

**Comment by andrewschoen** [comment]
jenkins test luminous-ansible2.2-update_docker_cluster

**Comment by andrewschoen** [comment]
jenkins test luminous-ansible2.2-purge_docker_cluster

**Comment by andrewschoen** [comment]
jenkins test luminous-ansible2.2-centos7_cluster

**Pull request merged**